webapp/channels/src/components/menu/menu.tsx

```diff
@@ -73,6 +73,9 @@ interface Props {
     menuButtonTooltip?: MenuButtonTooltipProps;
     menu: MenuProps;
     children: ReactNode[];
+
+    // Use MUI Anchor Playgroup to try various anchorOrigin
+    // and transformOrigin values - https://mui.com/material-ui/react-popover/#anchor-playground
```

Comment:
Is this comment still relevant?

Reply:
yes, its a necessary reference for understanding the meaning of the props. I was stuck for too long to admit :P without this reference links.